broker/core/scheduler-jobs/src/Scheduler.js

```diff
@@ -36,6 +36,10 @@ class Scheduler {
     this.jobs = {};
     this.time = new Date();
     this.config = config.scheduler;
+    if(!this.config) {
```

Comment:
This check is needed because `const { BaseService } = require('@sf/provisioner-services');` now loads `DirectorService` also. In the jobs such VirtualHostOperator/PostgresqlMTOperator the configs for scheduler is not specified and this causes error when DirectorService also tries to load Scheduler module.